packages/components/src/components/breadcrumb/breadcrumb.tsx

```diff
@@ -1,24 +1,29 @@
-import { Component, h, Prop } from '@stencil/core';
+import { Component, h, Element } from '@stencil/core';
 
 @Component({
   tag: 'ifx-breadcrumb',
   styleUrl: 'breadcrumb.scss',
   shadow: true,
 })
 export class Breadcrumb {
-  @Prop() breadcrumbs: { text: string, url: string }[];
+  @Element() el;
+
+  componentDidLoad() { 
+    const breadcrumbItems = this.el.querySelectorAll('ifx-breadcrumb-item')
+    for(let i = 1; i < breadcrumbItems.length; i++) { 
+      let label = breadcrumbItems[i].querySelector('ifx-breadcrumb-item-label')
```

Comment:
Synchronous Query in componentDidLoad method: Querying elements in componentDidLoad method using querySelector or querySelectorAll might lead to issues. The componentDidLoad method runs after the first render, but the issue is that the elements inside <slot /> may not be rendered yet when componentDidLoad is fired, as the slot content is provided by the component's user, not the component itself. This could lead to not selecting any elements.